### DISCUSSION TITLE
I made a Unity 3D SDK

### DISCUSSION THREAD

**Discussion by Sov3rain** [pocketbase/pocketbase#6021] [Show and tell]
Hey!

I really like this project and wanted to use it as a backend for a Unity app of mine. The only other option for Unity out there is Firebase, and the SDK is crap. I saw an opportunity and I ported the Dart SDK to Unity C#.

It's cross platform and written in fully managed C#. The only dependency is Newtonsoft.Json, as the built-in json parser in Unity is too limited. It's also built on top of UnityWebRequest to maximise platform compatibility.

If you want to give it a shot, it's available here: [PocketBase Unity SDK](https://github.com/Sov3rain/pocketbase-unity)

**Comment by Sov3rain** [comment]
Sorry, I doxxed myself, I need to clean the repo before making it public again.

**Reply by Sov3rain** [reply]
Done!

**Comment by redroofgames-tim** [comment]
I'm not sure if this is the place to write this, but I just wanted to say thank you! I've using your Unity SDK every day for the last few months, in a prototype project. It's been great so far – much appreciated!

**Reply by Sov3rain** [reply]
Hey, thank you very much! I opened [discussions](https://github.com/Sov3rain/pocketbase-unity/discussions) on my own repo, feel free to leave feedbacks on the SDK, much appreciated :)
